models/nulog/NuLogModel.py

```diff
@@ -206,10 +205,10 @@ def forward(self, query, key, value, mask=None):
         nbatches = query.size(0)
 
         # 1) Do all the linear projections in batch from d_model => h x d_k
-        query, key, value = [
+        query, key, value = (
```

Comment:
would changing from `array` to `tuple` expected though?

Reply:
makes no difference